packages/project/tsconfig.json

```diff
@@ -1,12 +1,18 @@
-// Extends can't use node's resolution rn https://github.com/dividab/tsconfig-paths/issues/209#issuecomment-1177261091
 {
   "extends": "../../node_modules/@webstudio-is/tsconfig/base.json",
   "include": ["./src/**/*.ts"],
   "compilerOptions": {
+    "composite": true,
     "module": "commonjs",
     "target": "ES2020",
     "strict": true,
     "baseUrl": ".",
-    "outDir": "./lib"
-  }
+    "outDir": "./lib",
+    "rootDir": "./src"
+  },
+  "references": [
```

Comment:
we should probably generate a comment here that these are generated and from where ... just in case someone tries to update these manually

Reply:
related: https://github.com/webstudio-is/webstudio-designer/pull/298#discussion_r1002433115